modules/core/packer/consul/site.yml

```diff
@@ -13,3 +14,11 @@
   - name: Install Consul
     include_role:
       name: "{{ playbook_dir }}/../roles/consul"
+  - name: Install CA Certificate
+    include_role:
+      name: "{{ playbook_dir }}/../roles/ansible-ca-store"
+    vars:
+      certificate: "{{ ca_certificate }}"
+      certificate_rename: "ca.crt"
+    become: yes
+    when: ca_certificate
```

Comment:
Just to verify, is `ca_certificate` false when it is an empty string?

Reply:
Yeah, in Python land:

```
In [2]: "" or "test"
Out[2]: 'test'
```

Reply:
:ok_hand: